plugin/loop/loop.go

```diff
@@ -49,7 +50,11 @@ func (l *Loop) ServeDNS(ctx context.Context, w dns.ResponseWriter, r *dns.Msg) (
 	}
 
 	if l.seen() > 2 {
-		log.Fatalf("Seen \"HINFO IN %s\" more than twice, loop detected", l.qname)
+		zone := "."
+		if len(dns.SplitDomainName(l.qname)) > 2 {
+			zone = dnsutil.Join(dns.SplitDomainName(l.qname)[2:]...)
```

Comment:
why is this not `l.zone`?

We should probably also point to https://coredns.io/plugins/loop#looping (and adjust loop/README.md to have a section on it under `#looping`)

Reply:
ha `l.zone` of course - the answer is already there.

> We should probably also point to

Do you mean put url in the error message? e.g. 

`Forwarding loop detected in "." zone. See https://coredns.io/plugins/loop#looping. ...`